pkg/clusterd/disk.go

```diff
@@ -108,7 +108,8 @@ func DiscoverDevices(executor exec.Executor) ([]*LocalDisk, error) {
 
 		fs, err := sys.GetDeviceFilesystems(d, executor)
 		if err != nil {
-			return nil, err
+			logger.Warningf("skipping device %s with an unknown filesystem. %+v", d, err)
```

Comment:
info level would be good, it doesn't seem like something the user needs to worry about

Reply:
this follows the other message severity.